### DISCUSSION TITLE
MaxAmount on site is different then when using API

### DISCUSSION THREAD

**Discussion by IntermidiateCoder** [cloudbet/docs#12] [Q&A]
Hey,
I've been using your API for a couple of months now and its great!
I have one question as I'm increasing my bet amounts over time.
Why is the max amount on site is different then when using API and which should I look at when betting through the API?

Example:
GMT v Singularity - Valorant match starts at 2PM UTC 31/08/2021
On site when clicking the MAX button when choosing winner GMT at odds 1.61 I get 0.02399ETH which is roughly 70euros.
But when looking at the same market and selection through API I get "max_amount": 1633.33 I'm assuming the number is in euros as specified in the API.

When I'm betting through the API I can bet up to 1633euros or 70euros?

Best Regards.

**Comment by kgravenreuth** [comment]
Hi @IntermidiateCoder ,
Thanks for providing details about the issue which you have observed while using the Cloudbet API.

The maximum stake which you can bet is determined by the `maxStake` field provided by our Feed API endpoints, and this should be similar to the maximum stake you observe on our site. Is this the field that you are referring to?

We have checked the event mentioned and we haven't observed any anomalies in the `maxStake` field since the values provided in the Feed API endpoint ( https://sports-api.cloudbet.com/pub/v2/odds/events/8837585 ) were similar to the maximum stake available on the site as per our investigation so far.

If you observe this issue again, could you please provide us the API response you received along with your Cloudbet username so that we can investigate in more detail? Please feel free to give details here or you can also send me private details via email klaus@cloudbet.com
